frontend/packages/core/src/NPS/header.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import {
+  ClickAwayListener,
+  Grow as MuiGrow,
+  MenuList,
+  Paper as MuiPaper,
+  Popper as MuiPopper,
+} from "@material-ui/core";
+import ChatBubbleOutlineIcon from "@material-ui/icons/ChatBubbleOutline";
+import { sortBy } from "lodash";
+
+import type { Workflow } from "../AppProvider/workflow";
+import { IconButton } from "../button";
+import { useAppContext } from "../Contexts";
+import type { SelectOption } from "../Input";
+import styled from "../styled";
+
+import NPSFeedback from "./feedback";
+
+const Grow = styled(MuiGrow)((props: { placement: string }) => ({
+  transformOrigin: props.placement,
+}));
+
+const Popper = styled(MuiPopper)({
+  padding: "0 12px",
+  marginLeft: "12px",
+  zIndex: 1201,
+});
+
+const Paper = styled(MuiPaper)({
+  width: "350px",
+  border: "1px solid #E7E7EA",
```

Comment:
I'm not seeing a border in figma but I might be missing it

Reply:
Leftover from copying from the user popout. Can't see a border anyways, so removing.